ext/pdo_pgsql/pgsql_driver.c

```diff
@@ -361,13 +361,17 @@ static char *pdo_pgsql_last_insert_id(pdo_dbh_t *dbh, const char *name, size_t *
 {
 	pdo_pgsql_db_handle *H = (pdo_pgsql_db_handle *)dbh->driver_data;
 	char *id = NULL;
+	char *version = NULL;
 
 	PGresult *res;
 	ExecStatusType status;
 	const char *q[1];
 	q[0] = name;
 
-	if (PHP_PDO_PGSQL_LASTVAL_PG_VERSION <= PQserverVersion && name == NULL) {
+	res = PQexec(H->server, "SHOW server_version_num");
+	version = estrdup((char *)PQgetvalue(res, 0, 0));
+
+	if (PHP_PDO_PGSQL_LASTVAL_PG_VERSION <= atoi(version) && name == NULL) {
```

Comment:
@weltling 
I added what you said and look the error now:

/home/travis/build/php/php-src/ext/pdo_pgsql/pgsql_driver.c:370:59: error: ‘pdo_pgsql_db_handle’ has no member named ‘servir’
make: *** [ext/pdo_pgsql/pgsql_driver.lo] Error 1